src/lib/datepicker/calendar-footer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import {Directionality} from '@angular/cdk/bidi';
+import {Component} from '@angular/core';
+import {async, ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import {MatNativeDateModule} from '@angular/material/core';
+import {MatDatepickerIntl} from './datepicker-intl';
+import {MatDatepickerModule} from './datepicker-module';
+
+describe('MatCalendarFooter', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        MatNativeDateModule,
+        MatDatepickerModule,
+      ],
+      declarations: [
+        // Test components.
+        StandardCalendar,
+      ],
+      providers: [
+        MatDatepickerIntl,
+        {provide: Directionality, useFactory: () => ({value: 'ltr'})},
```

Comment:
This isn't necessary, it'll default to `ltr` if a `Directionality` isn't provided.